test/test_spectral_ops.py

```diff
@@ -148,39 +145,32 @@ def test_fft_numpy(self, device, dtype):
             )
         ]
 
+        torch_fn = op.get_op()
+        numpy_fn = op.ref
 
-        fft_functions = ['fft', 'ifft', 'hfft', 'irfft']
-        # Real-only functions
-        if not dtype.is_complex:
-            fft_functions += ['rfft', 'ihfft']
-
-        for fname in fft_functions:
-            torch_fn = getattr(torch.fft, fname)
-            numpy_fn = getattr(np.fft, fname)
-
-            def fn(t: torch.Tensor, n: Optional[int], dim: int, norm: Optional[str]):
-                return torch_fn(t, n, dim, norm)
-            scripted_fn = torch.jit.script(fn)
+        def fn(t: torch.Tensor, n: Optional[int], dim: int, norm: Optional[str]):
```

Comment:
fyi @Lilyjjo is working on OpInfo PRs that will automatically test for variant consistency between eager mode and the JIT, and this test will be able to remove this comparison once it's landed (which should be very soon)

Reply:
I assume you're referring to the PR I had lots of conflicts with, so I've removed the script op.